recipe/meta.yaml

```diff
@@ -1,38 +1,56 @@
-{% set version = "3.0.0b2.post2" %}
+{% set version = "3.0.0b2" %}
+{% set version_str = "v3.0.0-beta-2" %}
 
 package:
     name: protobuf
     version: {{ version }}
 
 source:
-    fn: protobuf-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/p/protobuf/protobuf-{{ version }}.tar.gz
-    md5: 42aa7ab79ebcb94917e938ff482ef7a6
+    fn: protobuf_{{ version }}.tar.gz
+    url: https://github.com/google/protobuf/archive/{{ version_str }}/protobuf-{{ version_str }}.tar.gz
+    md5: e7f2602baffcbc27fb607de659cfbab6
 
 build:
-    number: 2
+    number: 3
+    skip: true  # [win]
 
 requirements:
     build:
+        - autoconf
+        - automake
+        - libtool
+        - pkg-config
         - python
-        - setuptools
+        - six
+        - ordereddict        # [py26]
+        - unittest2          # [py26]
+        - gcc                # [osx]
```

Comment:
Maybe. Doubtful though. My understanding from reading lots of issues like this that TR1 support in gcc 4.2.1 (Apple's gcc) is pretty weak. Most people end up picking a newer gcc to have better TR1 support. In the end, I think we want to switch to clang (TR1 is replaced with C++11), but protobuf is still trying to clean out TR1 or at least opt for C++11 first in several places.